dotnet/docusaurus.config.js

```diff
@@ -84,18 +84,22 @@ module.exports = {
             {
               label: '.NET',
               'data-language-prefix': '/dotnet/',
+              href: '#',
```

Comment:
Docusaurus requires now href/to we use `#` as a hack since we have our own logic in the `src/theme/NavbarItem/DefaultNavbarItem.js`